frontend/packages/dev-console/src/components/project-details/utils.ts

```diff
@@ -0,0 +1,16 @@
+import { connect } from 'react-redux';
+
+export const PROJECT_LIST_URI = '/k8s/cluster/projects';
+
+export const redirectURI = (ns) => `${PROJECT_LIST_URI}/${ns}`;
+
+export const useActiveNamespace = (Component) =>
+  connect((state: any) => {
+    const activeNamespace = state.UI.get('activeNamespace');
```

Comment:
Per @christianvogt's request, we have a selector we can use.

Found in `frontend/public/reducers/ui.ts`